src/babel/__tests__/unit.test.js

```diff
@@ -212,7 +212,7 @@ describe('babel plugin', () => {
       export const MyComponent = function () {
         const something = '';
         // comment
-        const [_Modal, _myGlobal] = di([Modal, myGlobal], MyComponent);
+        const [_Modal, _myGlobal] = di(MyComponent, Modal, myGlobal);
```

Comment:
side note: v8 dont like functions with variable arguments (causes deoptimization)

Better than creating arrays in place, but is not perfect. 
Wondering if we can use arrays pre-created outside of component's code